### PR TITLE
Exit non-zero when running sensuctl cmds that only have subcmds

### PR DESCRIPTION
**Pull request by amdprophet**
## What is this change?

Updates sensuctl commands that only contain subcommands to exit 46 (EX_USAGE from `sysexits.h`) when executed.

## Why is this change necessary?

See https://github.com/sensu/sensu-go/issues/3912.

## Does your change need a Changelog entry?

Yes.

## Do you need clarification on anything?

Nope!

## Were there any complications while making this change?

Cobra root commands behave differently than subcommands and thus the root command cannot return an error without displaying different output than subcommands.

## Have you reviewed and updated the documentation for this change? Is new documentation required?

No new documentation is required.

## How did you verify this change?

Tested by running:
* `sensuctl check` (exits 46, displays usage)
* `sensuctl check creat` (exits 46, displays suggestions)
* `sensusctl check create` (works as it did before)
* `sensuctl check help` (exits 0, displays usage)

## Is this change a patch?

No.

### CODE REVIEW

File: command/sysexits.go [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+	ExitNoPermission = 77
+
+	// Something was found in an unconfigured or misconfigured state.
+	ExitConfig = 78
+)
+
+type CommandErrorer interface {
+	ExitStatus()
+	Error()
+}
```
Comment (echlebek) [inline command/sysexits.go:77]: Not sure I understand what this is. Can it be removed?

File: command/sysexits.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
+package command
+
+// Based on sysexits.h from FreeBSD (also used by Linux)
+// https://www.freebsd.org/cgi/man.cgi?query=sysexits&manpath=FreeBSD+13.0-current
+
+var (
+	// The command executed successfully.
+	ExitOk = 0
```
Comment (echlebek) [inline command/sysexits.go:8]: Ideally this would be named ExitOK

File: cli/commands/helpers/subcommand.go [hunk 3]
```diff
@@ -0,0 +1,63 @@
+package helpers
```
Comment (echlebek) [inline cli/commands/helpers/subcommand.go:1]: I feel this code would be better namespaced in the command package, as "helpers" has less meaning. https://github.com/golang/go/wiki/CodeReviewComments#package-names

**Review by echlebek — COMMENTED** [review]
Love the docs on the exit constants, great work. I do have a few quibbles about names and I think I may have found some dead code.

File: command/sysexits.go (hunk 1 above)
Comment (amdprophet) [reply command/sysexits.go:77]: Good catch - I meant to use it here: https://github.com/sensu/sensu-go/blob/e47c19449cedeb33fbee0f7ad054e971aa3745d9/cmd/sensuctl/main.go#L25. Updating.

**Review by echlebek — APPROVED** [review]
Looks good, can you add a line to the changelog mentioning the help display fixes?

**Review by echlebek — APPROVED** [review]

**Pull request merged**